build.gradle

```diff
@@ -77,13 +81,15 @@ checkerFramework {
     'org.checkerframework.checker.nullness.NullnessChecker',
     'org.checkerframework.checker.regex.RegexChecker',
     'org.checkerframework.checker.signature.SignatureChecker',
-    'org.checkerframework.common.initializedfields.InitializedFieldsChecker'
+    'org.checkerframework.common.initializedfields.InitializedFieldsChecker',
+    'org.checkerframework.checker.objectconstruction.ObjectConstructionChecker',
   ]
   extraJavacArgs = [
     '-Werror',
     '-AcheckPurityAnnotations',
     '-ArequirePrefixInWarningSuppressions',
     '-AwarnUnneededSuppressions',
+    '-AcheckMustCall',
```

Comment:
If you also pass `-AdisableReturnsReceiver`, the Returns Receiver sub-checker will be disabled, which should make the Must Call Checker run faster (without increasing false positives)

Reply:
Passing `-AdisableReturnsReceiver` gives me `warning: The following options were not recognized by any processor: '[disableReturnsReceiver]'`; see https://github.com/mernst/plume-util/pull/3.

Passing `-ACalledMethodsChecker_disableReturnsReceiver` gives me
`warning: The following options were not recognized by any processor: '[CalledMethodsChecker_disableReturnsReceiver]'`; see https://github.com/mernst/plume-util/pull/4.

Did I make a mistake?

I also created a pull request to document the option: 
https://github.com/typetools/checker-framework/pull/4438.  However, given the failures above I don't trust that pull request.

Reply:
@kelloggm do you have any thoughts here?  One issue might be the option is missing from the `@SupportedOptions` annotation: 

https://github.com/typetools/checker-framework/blob/067f2ac4a19c85acb7d59cfcfa8e5c8a9ea6a183/checker/src/main/java/org/checkerframework/checker/calledmethods/CalledMethodsChecker.java#L26-L30

Reply:
This was an oversight when I wrote https://github.com/typetools/checker-framework/pull/3972. It wasn't caught by testing because the accompanying test case runs with `-Anowarn`. I updated Mike's PR (typetools/checker-framework#4438) to add the supported option and fix the test case.